assets/parquet/ingest_pairwise_arrays.py

```
@@ -0,0 +1,110 @@
+import pyarrow as pa
+import pyarrow.parquet as pq
+import json
+import sys
+import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)